students/1/main.js

```diff
@@ -31,12 +31,36 @@ const ex5 = async () => {
 }
 
 const ex6 = async () => {
-    console.log("TODO...")
+    let result = await getPostWithName(1); // 1 is the post id.
+    console.log(result);
 }
 
 //
 // Your functions here...
 //
+
+const getPostWithName = async (postId) => {
+    let endRes = 0;
+    let endSults = 0;
+    try {
+        let result = await axios.get(`https://jsonplaceholder.typicode.com/posts/${postId}`);
+        endRes = result.data
+        endRes.name = 'Leanne Graham'
+       
+
+        console.log(endRes.userId)
+
+        let userId = 1;
```

Comment:
We can't really do this, this is an example of hard coding and doesn't make our function work with other user ids.